scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -17,9 +17,9 @@ workflow GvsExtractCohortFromSampleNames {
     # not using the defaults in GvsPrepareCallset because we're using pre created datasets defined by the caller
     String destination_dataset_name
     String destination_project_id
+    String? fq_gvs_extraction_temp_tables_dataset
```

Comment:
Making this optional for now. AoU will likely stop providing this, and just let it go into the destination dataset.